src/bp-templates/bp-nouveau/buddypress-functions.php

```diff
@@ -120,6 +120,12 @@ function() {
 				},
 				0
 			);
+
+			// When BP Classic is activated, regular themes need this filter. 
+			if ( function_exists( 'bp_classic' ) ) {
+				// Set the BP Uri for the Ajax customizer preview.
+				add_filter( 'bp_uri', array( $this, 'customizer_set_uri' ), 10, 1 );
+			}
```

Comment:
I just committed this to your branch, this is the needed change before SVN commit it.